exporter/opentelemetry-exporter-otlp-proto-http/src/opentelemetry/exporter/otlp/proto/http/version.py

```diff
@@ -0,0 +1,15 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__version__ = "1.4.0.dev0"
```

Comment:
I suggest we mark this as 0.23.dev0 until the port discussion is sorted in the spec https://github.com/open-telemetry/opentelemetry-specification/issues/1816

Reply:
Issue on collector repo https://github.com/open-telemetry/opentelemetry-collector/issues/3617. I think it will take some to get the port fixed. I will change this to 0.23.dev0.